metadata.rb

```diff
@@ -5,9 +5,9 @@
 license          'Apache 2.0'
 description      'Installs Terraform (terraform.io)'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '0.4.4'
+version          '0.5.0'
 
-depends          'ark', '~> 1.0.0'
+depends          'ark', '~> 1.1'
```

Comment:
I've been of the school of thought stating that dependencies on 3rd party libraries should be considered riskier than those developed yourself or within one's close circle of friends. With that, the constraint on the ark dependency should be down to the patch level: `depends 'ark', '~> 1.1.0'`.